frontend/src/styles/_theme.scss

```diff
@@ -1,5 +1,5 @@
 $theme-dark: $blue-90;
-$theme-light: rgba(37, 0, 62, .8);
```

Comment:
@youwenliang changing this to the same value w/o an opacity layer since that made the gradient look stripe-y

Reply:
sure! sounds good